content/specs/at-uri-scheme.md

```diff
@@ -1,88 +1,133 @@
+
+- Authority part, preceded by double slash: supported
+- Empty authority part: not supported
+- Userinfo: not currently supported, but reserved for future use. a lone `@` character preceding a handle is not valid (eg, `at://@handle.example.com` is not valid)
+- Host and port separation: not supported. syntax conflicts with DID in authority part
+- Path part: supported, optional
+- Query: supported in general syntax, not currently used
+- Fragment: supported in general syntax, not currently used
+- Relative references: not yet supported
+- Normalization rules: supported in general syntax, not currently used
+
+Unfortunately, AT URIs are not compliant with the WHATWG URL Standard ([https://url.spec.whatwg.org/](https://url.spec.whatwg.org/)). Un-encoded colon characters in DIDs in the authority part of the URI are disallowed by that standard. Note that it is possible to un-ambigiously differentiate a DID in the authority section from a `host:port` pair. DIDs always have at least two colons, always begin with `did:`, and the DID method can not contain digits.
+
+### Full AT-URI Syntax
+
+The full syntax for AT URIs is flexible to a variety of future use cases, including future extensions to the path structure, query parameters, and a fragment part. The full syntax rules are:
+
+- The overall URI is restricted to a subset of ASCII characters
+- For reference below, the set of "unreserved characters", as defined in [RFC-3986](https://www.rfc-editor.org/rfc/rfc3986), includes alphanumeric (`A-Za-z0-9`), period, hyphen, underscore, and tilde (`.-_~`)
+- Maximum overall length is 8 KBytes (which may be shortened in the future)
+- Hex-encoding of characters is permitted (but in practice not necessary)
+- The URI scheme is `at`, and an authority part preceded with double slashes is always required, so the URI always starts `at://`
+- An authority section is required and must be non-empty. the authority can be either an atproto Handle, or a DID meeting the restrictions for use with atproto. note that the authority part can *not* be interpreted as a host:port pair, because of the use of colon characters (`:`) in DIDs. Colons and "unreserved" characters should not be escaped in DIDs, but other reserved characters (including `#`, `/`, `$`, `&`, `@`) must be escaped.
+    - Note that none of the current "blessed" DID methods for atproto allow these characters in DID identifiers
+- An optional path section may follow the authority. The path may contain multiple segments separated by a single slash (`/`). generic URI path normalization rules may be used
+- An optional query part is allowed, following generic URI syntax restrictions
+- An optional fragment part is allowed, using JSON Path syntax
```

Comment:
```suggestion
### Full AT URI Syntax

The full syntax for AT URIs is flexible to a variety of future use cases, including future extensions to the path structure, query parameters, and a fragment part. The full syntax rules are:

- The overall URI is restricted to a subset of ASCII characters
- For reference below, the set of unreserved characters, as defined in [RFC-3986](https://www.rfc-editor.org/rfc/rfc3986), includes alphanumeric (`A-Za-z0-9`), period, hyphen, underscore, and tilde (`.-_~`)
- Maximum overall length is 8 KB (which may be shortened in the future)
- Hex-encoding of characters is permitted (but in practice not necessary)
- The URI scheme is `at`, and an authority part preceded with double slashes is always required, so the URI always starts `at://`
- An authority section is required and must be non-empty. the authority can be either an atproto Handle, or a DID meeting the restrictions for use with atproto. note that the authority part can *not* be interpreted as a host:port pair, because of the use of colon characters (`:`) in DIDs. Colons and unreserved characters should not be escaped in DIDs, but other reserved characters (including `#`, `/`, `$`, `&`, `@`) must be escaped.
    - Note that none of the current "blessed" DID methods for atproto allow these characters in DID identifiers
- An optional path section may follow the authority. The path may contain multiple segments separated by a single slash (`/`). Generic URI path normalization rules may be used.
- An optional query part is allowed, following generic URI syntax restrictions
- An optional fragment part is allowed, using JSON Path syntax
```